app/controllers/viewing_party_controller.rb

```diff
@@ -16,12 +16,13 @@ def create
 
   private
 
-  def create_parties
+  def create_user_parties
     party = Party.create(movie_id: params[:movie_id], movie_title: params[:title], start_time: params[:start_time], date: params[:date], image_path: params[:image], duration: params[:duration])
     UserParty.create!(user_id: params[:user_id], party_id: party.id, host: true)
-    params[:invitees].each do |key, value|
-      if value == "1"
-        UserParty.create!(user_id: key.to_i, party_id: party.id, host: false)
+
+    params[:invitees].each do |user_id, checkbox_value|
+      if checkbox_value == "1"
+        UserParty.create!(user_id: user_id.to_i, party_id: party.id, host: false)
```

Comment:
Great update to the way we named our key and value to better match what this block is creating.